airbyte-integrations/connectors/source-iterable/unit_tests/test_streams.py

```diff
@@ -133,138 +148,3 @@ def test_campaigns_metrics_parse_response():
             {"data": {"a": 1, "b": 2, "d": 3}},
             {"data": {"a": 6, "c": 1, "d": 2}},
         ]
-
-
-def test_iterable_stream_parse_response():
-    stream = Lists(authenticator=None)
-    with responses.RequestsMock() as rsps:
-        rsps.add(
-            responses.GET,
-            "https://api.iterable.com/lists/getUsers?listId=100",
-            json={"lists": [{"id": 1}, {"id": 2}]},
-            status=200,
-            content_type="application/json",
-        )
-        resp = requests.get("https://api.iterable.com/lists/getUsers?listId=100")
-
-        records = stream.parse_response(response=resp)
-
-        assert list(records) == [{"id": 1}, {"id": 2}]
-
-
-def test_iterable_stream_backoff_time():
-    stream = Lists(authenticator=None)
-    assert stream.backoff_time(response=None) is None
-
-
-def test_iterable_export_stream_backoff_time():
-    stream = Users(authenticator=None, start_date="2019-10-10T00:00:00")
-    assert stream.backoff_time(response=None) is None
-
-
-@pytest.mark.parametrize(
-    "current_state,record_date,expected_state",
-    [
-        ({}, "2022", {"profileUpdatedAt": "2022-01-01T00:00:00+00:00"}),
-        ({"profileUpdatedAt": "2020-01-01T00:00:00+00:00"}, "2022", {"profileUpdatedAt": "2022-01-01T00:00:00+00:00"}),
-        ({"profileUpdatedAt": "2022-01-01T00:00:00+00:00"}, "2020", {"profileUpdatedAt": "2022-01-01T00:00:00+00:00"}),
-    ],
-)
-def test_get_updated_state(current_state, record_date, expected_state):
-    stream = Users(authenticator=None, start_date="2019-10-10T00:00:00")
-    state = stream.get_updated_state(
-        current_stream_state=current_state,
-        latest_record={"profileUpdatedAt": pendulum.parse(record_date)},
-    )
-    assert state == expected_state
-
-
-@responses.activate
-def test_stream_stops_on_401(mock_lists_resp):
-    # no requests should be made after getting 401 error despite the multiple slices
-    users_stream = ListUsers(authenticator=None)
-    responses.add(responses.GET, "https://api.iterable.com/api/lists/getUsers?listId=1", json={}, status=401)
-    slices = 0
-    for slice_ in users_stream.stream_slices(sync_mode=SyncMode.full_refresh):
-        slices += 1
-        _ = list(users_stream.read_records(stream_slice=slice_, sync_mode=SyncMode.full_refresh))
-    assert len(responses.calls) == 1
-    assert slices > 1
-
-
-@responses.activate
-def test_listuser_stream_keep_working_on_500():
```

Comment:
Why are we changing this behavior?

Reply:
Updated to test backoff policy on 500 error